stand/res/results

```diff
@@ -0,0 +1 @@
+wtf
```

Comment:
Are you sure it is not WIP?

Reply:
sorry, that folder should have been deleted

Reply:
ok, please ping me when it is fixed